java/java.lsp.server/vscode/package.json

```diff
@@ -1106,6 +1110,11 @@
 					"when": "viewItem =~ /class:oracle.developer.ContainerRepositoryItem/",
 					"group": "inline@9"
 				},
+				{
+					"command": "nbls.cloud.assets.buildPushNativeImage",
+					"when": "viewItem =~ /class:oracle.developer.ContainerRepositoryItem/",
+					"group": "oci@1000"
```

Comment:
Is this group/sorting key correct (`buildPushImage` is `inline@9`) ?

Reply:
The `buildPushImage` is inline button, but `buildPushNativeImage` is a context menu item.